server/app/views/applicant/AddressCorrectionBlockTemplate.html

```diff
@@ -5,75 +5,78 @@
   <body>
     <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
 
-    <form
-      role="main"
-      id="cf-block-form"
-      th:action="${confirmAddressAction}"
-      method="POST"
-      th:with="suggestions=${addressSuggestionGroup.getAddressSuggestions()},
+    <main role="main">
```

Comment:
**suggestion (non-blocking):** The native `<main>` element has a default landmark role of 'main', so you could remove `role="main"`.